airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/ReplicationActivityImpl.java

```diff
@@ -217,6 +225,26 @@ private static StandardSyncOutput reduceReplicationOutput(final ReplicationOutpu
     return standardSyncOutput;
   }
 
+  private static void traceReplicationSummary(final ReplicationAttemptSummary replicationSummary) {
+    if (replicationSummary != null) {
```

Comment:
Could flip this around to avoid some of this nesting:
```
if (replicationSummary == null) {
  return;
}
```

Reply:
Will fix